src/lucky/renderable.cr

```diff
@@ -142,31 +142,43 @@ module Lucky::Renderable
     file(path, content_type, disposition, filename, status.value)
   end
 
+  private def send_response(body : String, content_type : String, status : Int32? = nil) : Lucky::TextResponse
```

Comment:
Maybe rather than `send_response` we could do `text_response` so it is clear it is for text_responses? What do you think?

Reply:
I feel like `text_response` might get confused with `text` which is a response.... Also, while I'm in there, what's [this method](https://github.com/luckyframework/lucky/blob/master/src/lucky/renderable.cr#L153) for? Can we deprecate it? or is there a specific use?

Reply:
Yeah that's the old method. Can be deprecated. 

And yeah that could be confusing but `send_response` is kind of confusing since it doesn't send anything. So I'm not totally sure what to do here. Maybe we rename `text` to `plain_text` and then have `text_response`? I imagine that *might* clear things up but I'm not sure

Reply:
How about `send_text_response`? I would be ok renaming `text` too only because we use `text` on Pages, and I don't know if people ever confuse those too.

Reply:
Yeah I like that! `send_text_response` and `text` -> `plain_text` to make it clearer

Reply:
Also if renaming `text` -> `plain_text` the generator for `lucky gen.action.browser` needs to use change too